crates/trigger/src/lib.rs

```diff
@@ -161,6 +161,21 @@ pub struct HostComponentInitData {
     sqlite: Vec<String>,
 }
 
+impl HostComponentInitData {
+    /// Create an instance of `HostComponentInitData`.  `key_value_init_values`
+    /// will be added to the default key-value store; `sqlite_init_statements`
+    /// will be run against the default SQLite database.
+    pub fn new(
+        key_value_init_values: &[(String, String)],
+        sqlite_init_statements: &[String],
+    ) -> Self {
+        Self {
+            kv: key_value_init_values.to_vec(),
+            sqlite: sqlite_init_statements.to_vec(),
```

Comment:
A little more flexible (and equivalent for those specific argument types):

```suggestion
        key_value_init_values: impl Into<Vec<(String, String)>>,
        sqlite_init_statements: impl Into<Vec<String>>,
    ) -> Self {
        Self {
            kv: key_value_init_values.into(),
            sqlite: sqlite_init_statements.into(),
```

Reply:
Oh, of course - thanks!